File: OLSource1/v1/VS_officedev/codesnippet/CSharp/how-to--resize-bookmark-controls_3.cs
            Microsoft.Office.Tools.Word.Document vstoDocument =
                Globals.Factory.GetVstoObject(this.Application.ActiveDocument);


            vstoDocument.ActiveWindow.View.ShowBookmarks = true;
            vstoDocument.Paragraphs[1].Range.InsertParagraphBefore();

            Word.Range firstParagraph = vstoDocument.Paragraphs[1].Range;
            firstParagraph.Text = "123456789abcdefghijklmnopqrstuvwxyz";

            Microsoft.Office.Tools.Word.Bookmark sampleBookmark =
                vstoDocument.Controls.AddBookmark(firstParagraph, "bookmark1");
            sampleBookmark.Start = sampleBookmark.Start + 5;
            sampleBookmark.End = sampleBookmark.End - 5;

File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/how-to--resize-bookmark-controls_3.vb

        Dim VstoDocument As Microsoft.Office.Tools.Word.Document = _
            Globals.Factory.GetVstoObject(Me.Application.ActiveDocument)

        vstoDocument.ActiveWindow.View.ShowBookmarks = True
        vstoDocument.Paragraphs(1).Range.InsertParagraphBefore()

        Dim firstParagraph As Word.Range = vstoDocument.Paragraphs(1).Range
        firstParagraph.Text = "123456789abcdefghijklmnopqrstuvwxyz"

        Dim sampleBookmark As Microsoft.Office.Tools.Word.Bookmark = _
            vstoDocument.Controls.AddBookmark(firstParagraph, "bookmark1")
        sampleBookmark.Start = sampleBookmark.Start + 5
        sampleBookmark.End = sampleBookmark.End - 5